documentation/modules/rn-2.5.adoc

```diff
@@ -124,18 +124,14 @@ When migrating VMs that are installed with RHEL 9 as guest operating system from
 
 When adding an OVA provider, the error message `ConnectionTestFailed` may instantly appear, although the provider is created successfully. If the message does not disappear after a few minutes and the provider status does not move to `Ready`, this means that the `ova server pod creation` has failed. link:https://issues.redhat.com/browse/MTV-671[(MTV-671)]
 
-.OVA migration with multiple disks does not function as expected
```

Comment:
@anarnold97 I've just dropped this one as it's been an issue for just ~a month and we didn't get reports on that